### ISSUE TITLE
On Windows user session is not remembered

### ISSUE THREAD

**Issue by ErkoKnoll** [Azure/azure-mobile-apps-cordova-client#16]
Works fine on iOS and Android where it requires to log in only once per 60 days but on Windows it always prompts user to reenter their credentials when app is started. It seems that the refresh token is not remembered on Windows devices. Authentication type is Azure Active Directory.

**Comment by adrianhall** [comment]
Hi Erko,

You need to store the original auth token (in client.currentUser.mobileServiceAuthenticationToken) somewhere, then do a client directed refresh to get the new token.  This is completely supported in the current implementation - blog here: https://shellmonger.com/2016/04/13/30-days-of-zumo-v2-azure-mobile-apps-day-7-refresh-tokens/

**Comment by ErkoKnoll** [comment]
Hi,

I tried out your suggested approach and also checked out the log. I started storing client.currentUser.mobileServiceAuthenticationToken after authentication and before authentication I loaded it as such:
client.currentUser = {
         mobileServiceAuthenticationToken: loadedToken
}
(currentUser at that stage is undefined so I had to set it as object literal)

And then I did a client directed refresh by invoking /.auth/refresh like described in the blog but the response was 403 forbidden.

Am I doing something wrong here?

While the refresh token functionality works perfectly fine for iOS and Android under the hood may I ask why this manual intervention is required for Windows devices? And are there plans to have the automatic refresh token handling implemented for Windows in the future?
